lib/track.ts

```diff
@@ -128,4 +135,33 @@ export class TrackClient {
       `${this.trackRoot}/customers/${encodeURIComponent(customerId)}/devices/${encodeURIComponent(deviceToken)}`,
     );
   }
+
+  isValidIdType(input: string) {
+    return ["id", "email", "cio_id"].includes(input)
```

Comment:
This would be better extracted as an enum in `lib/utils.ts` like this:

```typescript
enum Identifier {
  ID = 'id',
  Email = 'email',
  CioId = 'cio_id',
}
```

Then, you can use that in this function like so:

```suggestion
    return Object.values(Identifier).includes(input)
```